### DISCUSSION TITLE
actions/setup-go@v4: errors while syncing using actions-sync

### DISCUSSION THREAD

**Discussion by gl72** [community/community#51944] [Actions]
### Select Topic Area

Bug

### Body

Hi all,
I have noticed that there  was a change by @IvanZosimov in the package.json yesterday on the actions/setup-go repository yesterday. Today, when we sync the repository to our GHE instance, we get the following error:
`error syncing repository actions/setup-go: failed to push to repo: https://<internal GHE URL>/actions/setup-go.git: malformed unpack status: 0024unpack index-pack abnormal exit`
Is it just a coincidence or there´s something weird going on?
Thanks in advance for your help!

**Comment by airtower-luna** [comment]
`failed to push to repo: https:///actions/setup-go.git`

That error message looks like there's a hostname missing in the repository URL.

**Reply by gl72** [reply]
hi @airtower-luna it was just a typo in the text I have amended it

**Comment by gl72** [comment]
EDIT: I resolved doing a manual sync pulling from github.com and pushing to our internal GHE.
